doc/topics/releases/oxygen.rst

```diff
@@ -88,11 +88,36 @@ environments (i.e. ``saltenvs``) have been added:
    ignore all tags and use branches only, and also to keep SHAs from being made
    available as saltenvs.
 
+Windows Installer: Changes to existing config handling
+------------------------------------------------------
+Behavior with existing configuration has changed. With previous installers the
+existing config was used and the master and minion id could be modified via the
+installer. It was problematic in that it didn't account for configuration that
+may be defined in the ``minion.d`` directory. This change gives you the option
+via a checkbox to either use the existing config with out changes or the default
+config using values you pass to the installer. If you choose to use the existing
+config then no changes are made. If not, the existing config is deleted, to
+include the ``minion.d`` directory, and the default config is used. A
+command-line switch (``/use-existing-config``) has also been added to control
+this behavior.
+
+Windows Installer: Multi-master configuration
+---------------------------------------------
+The installer now has the ability to apply a multi-master configuration either
+from the gui or the command line. The ``master`` field in the gui can accept
+either a single master or a comma-separated list of masters. The command-line
+switch (``/master=``) can accept the same.
+
+Windows Installer: Command-line help
+------------------------------------
+The Windows installer will now display command-line help when a help switch
+(``/?``) is passed.
+
 Salt Cloud Features
--------------------
+===================
 
 Pre-Flight Commands
-===================
+-------------------
```

Comment:
Same comment here. `------` is the heading, `======` is the sub-heading. See #42879 for more info.